ui/pages/confirm-encryption-public-key/confirm-encryption-public-key.container.js

```diff
@@ -21,13 +19,12 @@ import { getMostRecentOverviewPage } from '../../ducks/history/history';
 import { getNativeCurrency } from '../../ducks/metamask/metamask';
 import ConfirmEncryptionPublicKey from './confirm-encryption-public-key.component';
 
+// Removing usage of useNativeCurrencyAsPrimaryCurrency because component ConfirmEncryptionPublicKey is not
```

Comment:
Hmm I don't understand this comment. Could you clarify?

Reply:
My bad will update comment, this container is related to a component that is not being used